src/app/scripts/cac/control/cac-control-directions.js

```diff
@@ -239,6 +226,18 @@ CAC.Control.Directions = (function (_, $, moment, Control, Geocoder, Routing, Ty
         directionsListControl.hide();
     }
 
+    /**
+     * Wipe out user-set trip options and clear controls
+     */
+    function clearUserSettings() {
```

Comment:
The directions controller owns the typeahead controls cleared here; the home page doesn't have a way to refer to them directly.